cypher/cypher-docs/src/docs/dev/ql/administration/security/show-user-privileges-syntax.asciidoc

```diff
@@ -1,11 +0,0 @@
-.Command syntax
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/manage-privileges.adoc